.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,18 @@
+---
+name: ✈ Feature request
+about: Need help with configuring pytest-cov
```

Comment:
This is the same as Support Request. You probably meant something different here.